### PR TITLE
shopping-cart

### PR DESCRIPTION
**Pull request by Max223871**
did shopping cart

### CODE REVIEW

**Review by Max223871 — COMMENTED** [review]
fixed the view

File: app/models/user.rb
Comment (fanfanlulu) [inline app/models/user.rb:1]: In the schema, the cart tabel is called cart. It is suggested to follow the schema we have for the DB foundation

File: app/controllers/carts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    puts "Inside show action"
+    return unless current_user&.cart
+
+    @cart = current_user.cart
+    @cart_items = @cart.cart_items
+  end
+
+  def update
+    @cart = current_user.cart
+    product_id = params[:product_id]
```
Comment (fanfanlulu) [inline app/controllers/carts_controller.rb:12]: There is no product_id in any model found, the code here is not able to find any instances to run the update method here. the same mis goes for the params[:quantity]

**Review by fanfanlulu — COMMENTED** [review]
In MVC, the convention is to have the same name for model, controller and view. There are places in the code which did not follow the convention and they creates misconnection with the model and methods in controller will not be able to function.

**Pull request closed without merging**